config/routes.rb

```diff
@@ -38,11 +38,11 @@
   get 'ioby' => "legacy#ioby"
 
   get 'login' => "user_sessions#new",      :as => :login
-  delete 'logout' => "user_sessions#destroy", :as => :logout
+  get 'logout' => "user_sessions#destroy", :as => :logout
   post 'register' => 'users#create'
   get 'reset' => 'users#reset'
   post 'reset' => 'users#reset'
-  post 'reset/key/:key' => 'users#reset'
+  get 'reset/key/:key' => 'users#reset'
```

Comment:
I had to change this verb and the above to get in order to get them to work on my local machine. Since they aren't really the parts I had to work on I assume they are correct? And I will revert them back after I have finished.

Reply:
I think you fixed a latent bug related to the Rails 4 upgrade-- i've spent much of the day chasing a few of these which we missed when doing the `routes.rb` refactoring. Thank you!